api/open_saves.proto

```diff
@@ -430,6 +441,14 @@ message SortOrder {
   string user_property_name = 4;
 }
 
+message RecordResponse {
```

Comment:
To make this clear, this should be `GetMultiRecordsResponse`

Reply:
Good idea.